src/gazebo_mavlink_interface.cpp

```diff
@@ -1531,6 +1531,7 @@ void GazeboMavlinkInterface::handle_control(double _dt)
 #endif
 
         double err = current - target;
+        err = std::max(std::min(err, 0.2), -0.2);
```

Comment:
0.2 looks like a magic number.

Reply:
@Jaeyoung-Lim could you add a comment there why the 0.2 is necessary/how you got to it? Then we get this in right away I would say, as it clearly helps against the propeller shaking.

Reply:
@bkueng I added a parameter that you can set the maximum joint errors